controllers/glanceapi_controller.go

```diff
@@ -665,15 +666,15 @@ func (r *GlanceAPIReconciler) generateServiceConfigMaps(
 	using the Client Key provisioned on the Ceph cluster
 	**/
 	if gb == "rbd" {
-		templateParameters["CephClusterFSID"] = instance.Spec.CephBackend.CephClusterFSID
-		templateParameters["CephClusterMonHosts"] = instance.Spec.CephBackend.CephClusterMonHosts
-		templateParameters["CephClientKey"] = instance.Spec.CephBackend.CephClientKey
+		templateParameters["CephClusterFSID"] = instance.Spec.CephBackend.ClusterFSID
```

Comment:
By changing do you mean make those names consistent? (s/CephClusterFSID/ClusterFSID  and so forth)
In theory yes, we can do that, and we have occurrences in [1] we could change.

[1] https://github.com/openstack-k8s-operators/glance-operator/tree/master/templates/glanceapi/config

Reply:
Yes, that's the idea. I think having this additional mapping is unnecessary.

Reply:
+1, something like [1] should be enough.

[1] https://github.com/openstack-k8s-operators/cinder-operator/commit/d9209644d54ff35bf002a87007db28bc265d41f4